travis/push-to-cicd-status.sh

```diff
@@ -16,6 +16,10 @@ if [[ ${TRAVIS_REPO_SLUG##*/} != "acc-provision" ]]; then
 else
     PYPI_REGISTRY=$1
     TAG_NAME=$2
+    RELEASE="false"
+    if [ -n "$3" ]; then
```

Comment:
you are sending it from push-to-pypi.sh as "false" then why explicitly checking number of arguments. This should be definitive so comes as false for non acc-prov and true for acc-prov

Reply:
I think we can change the var name from RELEASE to like IS_RELEASE to convey that it's a check. The RELEASE var can be confused with the release name